one/src/events.rs

```diff
@@ -156,3 +383,236 @@ async fn migrate_from_database(input_ceramic_db: PathBuf, store: SQLiteBlockStor
     );
     result.unwrap()
 }
+
+/// We use the SQLiteRootStore as a local cache of the EthereumRootStore
+///
+/// The EthereumRootStore is the authoritative root store but it is also immutable  
+/// once the blocks are final.
+/// We only pull each txHash once and store the root, blockHash and timestamp.
+/// After that Time Events validation can be done locally.
+#[derive(Debug, Clone)]
+pub struct SQLiteRootStore {
+    pool: SqlitePool,
+}
+
+impl SQLiteRootStore {
```

Comment:
Can we please add comments to clarify what "root" we are talking about here?  Especially once we have self-anchoring we are likely to have multiple "roots" - the root of my local tree plus the root of the tree submitted to ethereum.